ci-operator/step-registry/ipi/deprovision/vsphere/diags/ipi-deprovision-vsphere-diags-commands.sh

```diff
@@ -56,6 +50,9 @@ function collect_diagnostic_data {
     echo "$(date -u --rfc-3339=seconds) - capture console image from $vm"    
     govc vm.console -vm.ipath="$vm" -capture "${vcenter_state}/${vmname}.png"
   done
+  first_vm=$(echo ${clustervms} | cut -d" " -f1)
+  target_hw_version=$(govc vm.info -json=true "${first_vm}" | jq -r .VirtualMachines[0].Config.Version)
+  echo "{\"hw_version\":  \"${target_hw_version}\", \"cloud\": \"${cloud_where_run}\"}" > "${ARTIFACT_DIR}/runtime-config.json"
```

Comment:
Would it be possible if we add the information for hw
-version and cloud in the job log file? so that it will be quicker to access especially when we access the following link which includes the logs for all the jobs.

https://prow.ci.openshift.org/prowjobs.js?omit=managedFields,decoration_config,pod_spec